index.html

```diff
@@ -0,0 +1,441 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <title>Document</title>
+  </head>
+  <body class="">
+    <div class="project-wrapper">
+      <!-- Header area fixed to the left sifed -->
+      <header class="header-container">
+        <div class="header-space">
+          <!-- Top of left bar -->
+          <div class="header-top-side">
+            <!-- Logo section -->
+            <div class="twitter-logo">
+              <a href="#">
+                <img src="./twitter-icon/twitter-bird-icon.svg" alt="" />
+              </a>
+            </div>
+
+            <!-- Menu area -->
+            <div class="left-menu-type">
+              <div class="menu-list-content">
+                <div>
+                  <a href="#">
+                    <img
+                      src="./twitter-icon/twitter-home-icon.svg"
+                      alt=""
```

Comment:
Having alts empty is not helpful for accessibility, these should have descriptions of the icons in them :)